SK.WebApp/src/services/UserService.ts

```diff
@@ -0,0 +1,39 @@
+import { anyEndpointKeys } from './endpoints';
```

Comment:
We've talked about this 😀
Please move this file to `src/loginToApi.ts`, suffix "Service" doesn't add anything in terms of clarity

Reply:
done